src/wp-includes/functions.php

```diff
@@ -8071,7 +8071,7 @@ function wp_cache_set_last_changed( $group ) {
 	 * @since 6.3.0
 	 *
 	 * @param string    $group         The cache group name.
-	 * @param int       $time          The new last changed time.
+	 * @param string    $time          The new last changed time (msec sec).
 	 * @param int|false $previous_time The previous last changed time. False if not previously set.
```

Comment:
Thanks Till, the second params type is also incorrect.

```suggestion
	 * @param string|false $previous_time The previous last changed time. False if not previously set.
```